### ISSUE TITLE
Accessboston additional icons

### ISSUE THREAD

**Issue by sebastianebarb** [CityOfBoston/design-team-workflow#1153]
- Proofpoint (which used to be called Wombat, which was a better name!) - which is a cybersecurity training tool. 
- e-Builder is an application to manage capital construction projects. So construction equipment or buildings etc would be the icons I can think of that would make sense
- Fleethub is for car share, so a car 
- Career Center.  It turns out the iCIMS app has 2 parts - what we connected too (what is now called 'Current Job Openings' on the Portal) and also Career Center.  This part of the app is what Hiring Managers log into to post positions, review resumes, etc. Deirdre could probably show it to you.

**Comment by rachelbraun** [comment]
New icon for ESS and HCM requested by Lauren Patrick (current ones can be found [here](https://drive.google.com/drive/u/1/folders/1j4RV6uBZ3mkHGotxEoGcRTGxt5byqpjQ))

"For ESS, employees are able to enter and check absences, view paychecks, view benefits, and update taxes and personal data. And they can do it anywhere and on any device so maybe the icon can show an outline of a smart phone or tablet with an icon of a clock and/or dollar sign with a finger touching the screen,  Or a hand holding the device, or profile pic looking at a screen..."

"For hcm, the core users include personnel officer, timekeepers, recruiters, payroll managers, and central office staff.  And the main function is to hire, keep a record of job, earnings and tax data, pay employees, administer health benefit, and a tool for position/budget mgmt. (and more, but at a very high level it is the hr info system for COB). The icon presented for HCM was good, i was just hopeful I could see other ideas."

**Comment by sebastianebarb** [comment]
I had requested a different icon for ESS and HCM back in February, but the request was missed (I'm sure there's a lot going on and especially with the transition of Rachel leaving)!   

Suggestions for ESS and HCM from previous email:
For ESS, employees are able to enter and check absences, view paychecks, view benefits, and update taxes and personal data. And they can do it anywhere and on any device so maybe the icon can show an outline of a smart phone or tablet with an icon of a clock and/or dollar sign with a finger touching the screen,  Or a hand holding the device, or profile pic looking at a screen... 

For hcm, the core users include personnel officer, timekeepers, recruiters, payroll managers, and central office staff.  And the main function is to hire, keep a record of job, earnings and tax data, pay employees, administer health benefit, and a tool for position/budget mgmt. (and more, but at a very high level it is the hr info system for COB). The icon presented for HCM was good, i was just hopeful I could see other ideas. 

I'm sure it's too late for the initial launch, but is it possible to update the icons at a later date?  Also - I requested access to the google page with my personal email by accident - just ignore it. 

Thanks,
Lauren
